### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by rsmmr** [zeek/btest#72]
I've started to see these deprecation warnings:

```
/opt/homebrew/bin/btest:2298: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  v_now = distutils.version.LooseVersion(VERSION)
/opt/homebrew/bin/btest:2299: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  v_min = distutils.version.LooseVersion(getOption("MinVersion", None))
```

This is with Python 3.9 from Homebrew.

**Comment by ckreibich** [comment]
This might be us using an older `btest` somewhere ... the warning looks like a duplicate of #62, fixed by #63.

**Comment by ckreibich** [comment]
Robin I'll go ahead and close this, and of course feel free to reopen if you still see it. Perhaps it was an older `release/` branch.
